website/server/controllers/api-v3/content.js

```diff
@@ -1,11 +1,16 @@
 import nconf from 'nconf';
 import { langCodes } from '../../libs/i18n';
-import { CONTENT_CACHE_PATH, getLocalizedContentResponse } from '../../libs/content';
+import { serveContent } from '../../libs/content';
 
 const IS_PROD = nconf.get('IS_PROD');
 
 const api = {};
 
+const MOBILE_FILTER = `achievements,questSeriesAchievements,animalColorAchievements,animalSetAchievements,stableAchievements,
```

Comment:
I think it would be easier to maintain if this would use an array as source which then at the end used `.join(',')` to get the filter complete